### ISSUE TITLE
@-moz-document url-prefix()     fails 

### ISSUE THREAD

**Issue by athulvraj** [ortic/css2less#15]
@-moz-document url-prefix() {
  #timeSheetMainTable, .accountingBox {
     border-right: none;
  }
}
@-moz-document url-prefix() {
  .primaryButton, .formButton {line-height:23px;}
}

**Comment by Remo** [comment]
Yes, thanks! It's probably the CSS parser I'm using which doesn't support this )-:
Will have a look
